### PR TITLE
Adding a Next() function to grpc client

### PR DESCRIPTION
**Pull request by AnomalRoil**
This will be useful in the coming months, hopefully.

Also, updating to the released v2.0.2 instead of using outdated pre-releases.

### CODE REVIEW

File: grpc/grpc.go [hunk 1]
```diff
@@ -30,7 +29,10 @@ func init() {
 	balancer.Register(NewFallbackBuilder(FallbackTimeout))
 	// registers the logging_pick_first_with_fallback balancer
 	balancer.Register(NewLoggingBalancerBuilder("pick_first_with_fallback", slog.With("service", "balancer")))
-	bindMetrics()
+	err := bindMetrics()
+	if err != nil {
```
Comment (CluEleSsUK) [inline grpc/grpc.go]:
```suggestion
	if err := bindMetrics(); err != nil {
```

File: grpc/grpc.go [hunk 2]
```diff
@@ -77,7 +79,7 @@ func NewClient(serverAddr string, l logger) (*Client, error) {
 		grpc.WithTransportCredentials(insecure.NewCredentials()),
 		grpc.WithChainUnaryInterceptor(
 			// TODO: do we really want a 500ms default healthTimeout on unary RPC?
-			timeout.UnaryClientInterceptor(500*time.Millisecond),
+			//timeout.UnaryClientInterceptor(500*time.Millisecond),
```
Comment (CluEleSsUK) [inline grpc/grpc.go]:
```suggestion
```

File: routes_handler.go [hunk 3]
```diff
@@ -328,6 +328,35 @@ func GetLatest(c *grpc.Client, isV2 bool) func(http.ResponseWriter, *http.Reques
...
+	return func(w http.ResponseWriter, r *http.Request) {
+		m, err := createRequestMD(r)
+		if err != nil {
+			slog.Error("[GetNext] unable to create metadata for request", "error", err)
+			http.Error(w, "Failed to get latest", http.StatusInternalServerError)
+			return
+		}
+
+		beacon, err := c.Next(r.Context(), m)
+		if err != nil {
```
Comment (CluEleSsUK) [inline routes_handler.go:339]: nesting these looks like a mistake

Comment (AnomalRoil) [reply routes_handler.go:339]: What do you mean?

File: grpc/grpc.go [hunk 4]
```diff
@@ -77,7 +79,7 @@ func NewClient(serverAddr string, l logger) (*Client, error) {
 		grpc.WithTransportCredentials(insecure.NewCredentials()),
 		grpc.WithChainUnaryInterceptor(
 			// TODO: do we really want a 500ms default healthTimeout on unary RPC?
```
Comment (AnomalRoil) [inline grpc/grpc.go]:
```suggestion
```

File: routes_handler.go (hunk 3 above)
Comment (AnomalRoil) [reply routes_handler.go:339]: Ah seen it xD Yeah

**Pull request merged**